### PR TITLE
Project commit

### PR DESCRIPTION
**Pull request by yunusemrepekguc**
Project merge to main branch from dev

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+	    android:usesCleartextTraffic="true"
```
Comment (oguzozsoy) [inline app/src/main/AndroidManifest.xml]: Please clear whitespace.

File: smart_mediation/build.gradle [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    id 'kotlin-android-extensions'
+}
+
+android {
+    compileSdkVersion 31
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        applicationId "com.hmscl.huawei.smart_mediation"
+        minSdkVersion 23
```
Comment (oguzozsoy) [inline smart_mediation/build.gradle]: Is it possible to decrease the min SDK version?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (oguzozsoy) [inline app/src/main/AndroidManifest.xml]: Please check domain based clear text traffic.

File: smart_mediation/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.hmscl.huawei.smart_mediation">
+
+    <application
```
Comment (oguzozsoy) [inline smart_mediation/src/main/AndroidManifest.xml]: is it necessary?

**Review by oguzozsoy — COMMENTED** [review]
Is it possible to remove the application tag from the manifest and maybe we can add a clear text traffic configuration to the readme and reduce min SDK?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (BerkeCoban) [reply app/src/main/AndroidManifest.xml]: this line is removed.

File: smart_mediation/build.gradle (hunk 2 above)
Comment (BerkeCoban) [reply smart_mediation/build.gradle]: Clear text traffic is removed from the manifest, min SDK decreased to 19, (minimal requirement for Smart Display SDK and Huawei Ads SDK )

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (BerkeCoban) [reply app/src/main/AndroidManifest.xml]: This line will be added by developers in their app manifest if needed.

File: smart_mediation/src/main/AndroidManifest.xml (hunk 3 above)
Comment (BerkeCoban) [reply smart_mediation/src/main/AndroidManifest.xml]: application tag is removed from module's manifest, it is not necessary

**Pull request merged**